### PR TITLE
Temporarily remove shared_preferences dependency until it can be used with nnbd

### PR DESCRIPTION
**Pull request by rami-a**
This is to unblock the NNBD migration for flutter

### CODE REVIEW

File: lib/pages/demo.dart [hunk 1]
```diff
@@ -132,12 +132,14 @@ class _GalleryDemoPageState extends State<GalleryDemoPage>
...
     );
-    SharedPreferences.getInstance().then((preferences) {
-      setState(() {
-        _demoViewedCount = preferences.getInt(_demoViewedCountKey) ?? 0;
-        preferences.setInt(_demoViewedCountKey, _demoViewedCount + 1);
-      });
-    });
+    // TODO(rami-a): Re-enable once shared_preferences is migrated to NNBD.
+    _demoViewedCount = 10;
+//    SharedPreferences.getInstance().then((preferences) {
```
Comment (JoseAlba) [inline lib/pages/demo.dart:137]: Are we keeping the commented out code ?

**Review by JoseAlba — APPROVED** [review]
```
██╗      ██████╗ ████████╗███╗   ███╗
██║     ██╔════╝ ╚══██╔══╝████╗ ████║
██║     ██║  ███╗   ██║   ██╔████╔██║
██║     ██║   ██║   ██║   ██║╚██╔╝██║
███████╗╚██████╔╝   ██║   ██║ ╚═╝ ██║
╚══════╝ ╚═════╝    ╚═╝   ╚═╝     ╚═╝
```

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

File: lib/pages/demo.dart (hunk 1 above)
Comment (rami-a) [reply lib/pages/demo.dart]: Done

**Pull request merged**